mantle/kola/tests/ignition/systemd.go

```diff
@@ -56,5 +56,5 @@ func init() {
 func enableSystemdService(c cluster.TestCluster) {
 	m := c.Machines()[0]
 
-	c.AssertCmdOutputContains(m, "systemctl status nfs-server.service", "inactive")
```

Comment:
Hmm, don't we need something stronger here since `active` is a substring of `inactive`?

Reply:
Good point!  Fixed.